templates/base.html

```diff
@@ -37,6 +37,17 @@
   <meta name="MobileOptimized" content="320">
   <meta name="viewport" content="initial-scale=1">
 
+  <!-- Open Graph -->
+  {% set image = post.config.image[0].src if post.config.image else '/static/images/blog/oddsite.jpg' %}
+  <meta property="og:image" content="{{ config.canonical_url }}static/images/{% block og_image %}{{ image }}{% endblock %}" />
+  <meta property="og:url" content="{{ config.canonical_url }}{% block og_url %}{% endblock %}" />
+  <meta property="og:title" content="{% block og_title %}{{ post.title }}{% endblock %}" />
+  <meta property="og:description" content="{% block og_description %}{{ post.summary }}{% endblock %}" />
```

Comment:
I think line-breaks and rst markup were causing problems in my miriamsuzanne.com summary output. Any way to strip those things?